libs/geospatial/src/main/java/org/codice/ddf/libs/geo/conversion/DecimalDegreesConverter.java

```diff
@@ -0,0 +1,320 @@
+            double longitude = ddCoord.getLongitudeDegrees();
+
+            setVariables(latitude, longitude);
+
+            int longZone = getLongZone(longitude);
+            LatZones latZones = new LatZones();
+            char latZone = latZones.getLatZone(latitude);
+
+            double easting = getEasting();
+            double northing = getNorthing(latitude);
+
+            return new UtmCoordinate(longZone, latZone, easting, northing);
+
+        }
+
+        /**
+         * Converts the decimal degree to radians
+         *
+         * @param degree the decimal degree to convert
+         * @return the coordinate value in radians
+         */
+        private double degreeToRadian(double degree) {
+            return degree * Math.PI / 180;
+        }
+
+        /**
+         * Sets all the variables for doing the conversion to UTM
+         *
+         * @param latitude  the latitude value (in decimal degree format) to convert
+         *                  to UTM
+         * @param longitude the longitude value (in decimal degree format) to convert
+         *                  to UTM
+         */
+        private void setVariables(double latitude, double longitude) {
+            latitude = degreeToRadian(latitude);
+            nu = equatorialRadius / Math.pow(1 - Math.pow(e * Math.sin(latitude), 2), (1 / 2.0));
+
+            double var1;
+            if (longitude < 0.0) {
+                var1 = ((int) ((180 + longitude) / 6.0)) + 1;
+            } else {
+                var1 = ((int) (longitude / 6)) + 31;
+            }
+            double var2 = (6 * var1) - 183;
+            double var3 = longitude - var2;
+            p = var3 * 3600 / 10000;
+
+            s = a0 * latitude - b0 * Math.sin(2 * latitude) + c0 * Math.sin(4 * latitude)
+                    - d0 * Math.sin(6 * latitude) + e0 * Math.sin(8 * latitude);
+
+            k1 = s * k0;
+            k2 = nu * Math.sin(latitude) * Math.cos(latitude) * Math.pow(sin1, 2) * k0 * (100000000)
+                    / 2;
+            k3 = ((Math.pow(sin1, 4) * nu * Math.sin(latitude) * Math.pow(Math.cos(latitude), 3))
+                    / 24) * (5 - Math.pow(Math.tan(latitude), 2) + 9 * e1sq * Math.pow(Math.cos(
+                    latitude), 2) + 4 * Math.pow(e1sq, 2) * Math.pow(Math.cos(latitude), 4)) * k0
+                    * (10000000000000000L);
+
+            k4 = nu * Math.cos(latitude) * sin1 * k0 * 10000;
+
+            k5 = Math.pow(sin1 * Math.cos(latitude), 3) * (nu / 6) * (
+                    1 - Math.pow(Math.tan(latitude), 2) + e1sq * Math.pow(Math.cos(latitude), 2))
+                    * k0 * 1000000000000L;
+        }
+
+        /**
+         * Computes the Longitude zone (UTM Zone)
+         *
+         * @param longitude the logitude value (in decimal degree format) to convert
+         *                  the UTM Zone for.
+         * @return
+         */
+        private int getLongZone(double longitude) {
```

Comment:
This logic does not hold for all zones. There are extended zones in certain latitudes around Norway and Svalbard. See http://earth-info.nga.mil/GandG/publications/tm8358.1/tr83581b.html#ZZ28